src/components/Publish/_validation.ts

```diff
@@ -51,16 +85,34 @@ const validationService = {
   links: Yup.array<FileInfo[]>()
     .of(
       Yup.object().shape({
-        url: Yup.string()
-          .url('Must be a valid URL.')
-          .test(
-            'GoogleNotSupported',
-            'Google Drive is not a supported hosting service. Please use an alternative.',
-            (value) => {
-              return !value?.toString().includes('drive.google')
-            }
-          ),
-        // TODO: require valid file only when URL is given
+        url: Yup.string().test((value, context) => {
+          const { type } = context.parent
+          let validField
```

Comment:
wouldn't it be useful to have a helper to validate, and use it for both publish and edit, we can reduce code duplication like so

Reply:
done. Can you please check one last time, to make sure it's all fine 👍🏽 thanks bud